### PR TITLE
CS-43987-added stack name to the config

### PR DESCRIPTION
**Pull request by cs-raj**
version bump:
Already done in #1300  and will go in same release

### CODE REVIEW

File: packages/contentstack-import/src/import/module-importer.ts [hunk 1]
```diff
@@ -23,6 +23,8 @@ class ModuleImporter {
   }
 
   async start(): Promise<any> {
+    let stack_name:Record<string,unknown> = await this.stackAPIClient.fetch();
+    this.importConfig.stackName = stack_name.name as string;
```
Comment (aman19K) [inline packages/contentstack-import/src/import/module-importer.ts:27]: @cs-raj can you pls use camelcase here?

File: packages/contentstack-import/src/commands/cm/stacks/import.ts [hunk 2]
```diff
@@ -132,7 +132,7 @@ export default class ImportCommand extends Command {
       const result = await moduleImporter.start();
 
       if (!result?.noSuccessMsg) {
-        log(importConfig, `The content has been imported to the stack ${importConfig.apiKey} successfully!`, 'success');
+        log(importConfig, `The content has been imported to the stack named '${importConfig.stackName}' having api_key '${importConfig.apiKey}' successfully!`, 'success');
       }
```
Comment (aman19K) [inline packages/contentstack-import/src/commands/cm/stacks/import.ts:136]: @cs-raj , pls check text with doc team.

File: packages/contentstack-import/src/import/module-importer.ts (hunk 1 above)
Comment (cs-raj) [reply packages/contentstack-import/src/import/module-importer.ts]: Yes done

File: packages/contentstack-import/src/commands/cm/stacks/import.ts (hunk 2 above)
Comment (cs-raj) [reply packages/contentstack-import/src/commands/cm/stacks/import.ts]: Ui text verified and updated

**Review by aman19K — APPROVED** [review]

**Review by aman19K — APPROVED** [review]

File: packages/contentstack-import/src/import/module-importer.ts [hunk 3]
```diff
@@ -23,6 +23,8 @@ class ModuleImporter {
   }
 
   async start(): Promise<any> {
+    const stackName: Record<string, unknown> = await this.stackAPIClient.fetch();
+    this.importConfig.stackName = stackName.name as string;
```
Comment (antonyagustine) [inline packages/contentstack-import/src/import/module-importer.ts:27]: @cs-raj Can you validate this if it works fine with the management token?

**Review by antonyagustine — APPROVED** [review]

File: packages/contentstack-import/src/import/module-importer.ts (hunk 3 above)
Comment (cs-raj) [reply packages/contentstack-import/src/import/module-importer.ts]: Validated, the code threw an error, hence removed the condition to use stack name if management token is used

**Review by aman19K — APPROVED** [review]

**Review by antonyagustine — APPROVED** [review]

**Pull request merged**